packages/scaffolding-cli/templates/src/csr/README.md

```diff
@@ -47,3 +47,68 @@ https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
 
 This section has moved here:
 https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+
+## Cypress: Functional Testing
+
+## Getting started
+
+Run `npm install --save-dev cypress axe-core cypress-axe @types/cypress-axe @applitools/eyes-cypress`
+
+## Folder structure
+
+├── __tests__
+│   └── fixtures
+│     ├── add-menu-response.json
```

Comment:
Shouldn't these be generic names?

Reply:
Why do you say that? these fixures are stubbing out the menu calls made by the app.

Reply:
I get it. But these are meant to be abstract right. Not a big thing.

Reply:
You can't abstract stub fixtures when the the webapp is expecting it?

Reply:
Done in PR https://github.com/amido/stacks-webapp-template/pull/160